docs/diagrams/ModelClassDiagram.puml

```diff
@@ -13,12 +13,18 @@ Class ModelManager
 Class UserPrefs
 
 Class UniquePersonList
+Class UniqueEventList
 Class Person
 Class Address
 Class Email
 Class Name
 Class Phone
 Class Tag
+Class Event
+Class EventName
+Class Time
+Class Venue
+Class Contacts
```

Comment:
Should this be Person instead and the contact relationship is highlight by association label or something?